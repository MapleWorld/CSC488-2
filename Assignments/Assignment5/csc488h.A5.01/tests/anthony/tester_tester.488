begin
    integer x
    get x
    put x, skip, "That's all folks!"
end
