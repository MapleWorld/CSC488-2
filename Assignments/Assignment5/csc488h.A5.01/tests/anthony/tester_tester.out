50
That's all folks!
